sdk/cosmos/azure-cosmos-benchmark/ctl/linkedin/run_benchmark.sh

```diff
@@ -125,16 +132,16 @@ if [ -z "$ctl_graphite_endpoint" ]
 then
     if [ -z "$ctl_manage_database" ]
     then
-        java -Xmx$heap_size -Xms$heap_size  $jvm_opt -Dcosmos.directModeProtocol=$protocol -Dazure.cosmos.directModeProtocol=$protocol -jar "$jar_file" -serviceEndpoint "$service_endpoint" -masterKey "$master_key" -databaseId "$database_name"  -collectionId "$collection_name" -numberOfCollectionForCtl "$number_Of_collection" -throughput $throughput  -consistencyLevel $consistency_level -concurrency $concurrency -numberOfOperations $number_of_operations -operation $operation -connectionMode $connection_mode -maxRunningTimeDuration $max_running_time_duration -numberOfPreCreatedDocuments $number_of_precreated_documents -bulkloadBatchSize $bulk_load_batch_size -printingInterval $printing_interval -environment $ctl_environment 2>&1 | tee -a "$log_filename"
+        java -Xmx$heap_size -Xms$heap_size  $jvm_opt -Dcosmos.directModeProtocol=$protocol -Dazure.cosmos.directModeProtocol=$protocol -jar "$jar_file" -serviceEndpoint "$service_endpoint" -masterKey "$master_key" -databaseId "$database_name"  -collectionId "$collection_name" -numberOfCollectionForCtl "$number_Of_collection" -throughput $throughput  -consistencyLevel $consistency_level -concurrency $concurrency -numberOfOperations $number_of_operations -operation $operation -connectionMode $connection_mode -maxRunningTimeDuration $max_running_time_duration -numberOfPreCreatedDocuments $number_of_precreated_documents -bulkloadBatchSize $bulk_load_batch_size -printingInterval $printing_interval -environment $environment -testScenario $test_scenario 2>&1 | tee -a "$log_filename"
```

Comment:
$ctl_environment ->$environment is the intentional change right ?

Reply:
Yes it is. The current code above has this block, and that was an incorrect

if [ -z "$ctl_environment" ]
then
environment=Daily
else
environment=$ctl_environment
fi